templates/patterns/list_item/list_item.ui_patterns.yml

```diff
@@ -59,6 +59,14 @@ list_item:
       label: "Primary image"
       description: "Attributes of the image to render"
       preview:
-        src: "http://via.placeholder.com/400x400"
+        src: "https://placeimg.com/1200/900/arch"
         alt: "Alternative text for image"
-
+    infos:
+      type: "array"
+      label: "Infos"
+      description: "A list of infos of the page."
+      preview:
+        - text: "Munich, Germany"
+          icon: "location"
+        - text: "Live stream available"
+          icon: "livestreaming"
```

Comment:
On the "Content item" composition these are called "Secondary meta", ideally we could create patterns for these, so to reuse as much as we can. Would it be too bad to split this out in a follow up issue? So we can give it the right amount of attention?